src/ser.rs

```diff
@@ -194,9 +220,76 @@ impl<'a> Serializer<'a> {
         Serializer {
             dst: dst,
             state: State::End,
+            settings: Settings::default(),
         }
     }
 
+    /// Instantiate a "pretty" formatter
+    ///
+    /// By default this will use :
+    /// - pretty strings
+    /// - arrays with an indentation of 4 and a trailing comma
+    pub fn pretty(dst: &'a mut String) -> Serializer<'a> {
+        Serializer {
+            dst: dst,
+            state: State::End,
+            settings: Settings {
+                array: Some(ArraySettings::pretty()),
+                pretty_string: true,
+            },
+        }
+    }
+
+    /// Enable or Disable pretty strings
```

Comment:
Mind expanding this comment and those below to explain what "pretty" means in the context of arrays/strings?